code/modules/religion/religion_sects.dm

```diff
@@ -0,0 +1,157 @@
+
+/// Adjust Favor by a certain amount. Can provide optional features based on a user. Returns actual amount added/removed
+/datum/religion_sect/proc/adjust_favor(amount = 0, mob/living/L)
+	. = amount
+	if(favor + amount < 0)
+		. = favor //if favor = 5 and we want to subtract 10, we'll only be able to subtract 5
+	if((favor + amount > max_favor))
+		. = (max_favor-favor) //if favor = 5 and we want to add 10 with a max of 10, we'll only be able to add 5
+	favor = clamp(0,max_favor, favor+amount)
+
+/// Sets favor to a specific amount. Can provide optional features based on a user.
+/datum/religion_sect/proc/set_favor(amount = 0, mob/living/L)
+	favor = clamp(0,max_favor,amount)
+	return favor
+
+/// Activates when an individual uses a rite. Can provide different/additional benefits depending on the user.
+/datum/religion_sect/proc/on_riteuse(mob/living/user, obj/structure/altar_of_gods/AOG)
+
+/// Replaces the bible's bless mechanic. Return TRUE if you want to not do the brain hit.
+/datum/religion_sect/proc/sect_bless(mob/living/L, mob/living/user)
+	if(!ishuman(L))
+		return FALSE
+	var/mob/living/carbon/human/H = L
+	for(var/X in H.bodyparts)
+		var/obj/item/bodypart/BP = X
+		if(BP.status == BODYPART_ROBOTIC)
+			to_chat(user, "<span class='warning'>[GLOB.deity] refuses to heal this metallic taint!</span>")
+			return TRUE
+
+	var/heal_amt = 10
+	var/list/hurt_limbs = H.get_damaged_bodyparts(1, 1, null, BODYPART_ORGANIC)
+
+	if(hurt_limbs.len)
+		for(var/X in hurt_limbs)
+			var/obj/item/bodypart/affecting = X
+			if(affecting.heal_damage(heal_amt, heal_amt, null, BODYPART_ORGANIC))
+				H.update_damage_overlays()
+		H.visible_message("<span class='notice'>[user] heals [H] with the power of [GLOB.deity]!</span>")
+		to_chat(H, "<span class='boldnotice'>May the power of [GLOB.deity] compel you to be healed!</span>")
+		playsound(user, "punch", 25, TRUE, -1)
+		SEND_SIGNAL(H, COMSIG_ADD_MOOD_EVENT, "blessing", /datum/mood_event/blessing)
+	return TRUE
+
+/datum/religion_sect/puritanism
+	name = "Puritanism (Default)"
+	desc = "Nothing special."
+	convert_opener = "Your run-of-the-mill sect, there are no benefits or boons associated. Praise normalcy!"
+
+/// SECT_TECH
+/datum/religion_sect/technophile
+	name = "Technophile"
+	desc = "A sect oriented around technology."
+	convert_opener = "May you find peace in a metal shell, acolyte.<br>Bibles now recharge cyborgs and heal robotic limbs if targeted, but they do not heal organic limbs. You can now sacrifice cells, with favor depending on their charge."
+	alignment = ALIGNMENT_NEUT
+	desired_items = list(/obj/item/stock_parts/cell)
+	rites_list = list(/datum/religion_rites/synthconversion)
+	altar_icon_state = "convertaltar-blue"
+
+/datum/religion_sect/technophile/sect_bless(mob/living/L, mob/living/user)
+	if(iscyborg(L))
+		var/mob/living/silicon/robot/R = L
+		var/charge_amt = 50
+		if(L.mind?.holy_role == HOLY_ROLE_HIGHPRIEST)
+			charge_amt *= 2
+		R.cell?.charge += charge_amt
```

Comment:
can we make this recharge ethereals too (but still allow it to heal the robotic bodyparts of ethereals?)

Reply:
probably ill look at the code